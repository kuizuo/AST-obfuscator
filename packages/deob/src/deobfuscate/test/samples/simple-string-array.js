const arr = ['log', 'Hello, World!']
console[arr[0]](arr[1])

// ignore mutable array
const arr2 = ['log', 'Hello, World!']
arr2[0] = 'warn'
console[arr2[0]](arr2[1])
