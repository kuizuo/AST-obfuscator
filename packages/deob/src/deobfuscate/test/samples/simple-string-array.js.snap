console.log("Hello, World!");

// ignore mutable array
const arr2 = ["log", "Hello, World!"];
arr2[0] = "warn";
console[arr2[0]](arr2[1]);